### PR TITLE
log property name in counter for empty CSV cell values

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 1]
```diff
@@ -526,7 +555,8 @@ public static SchemaTerm parseSchemaTerm(String value, BiConsumer<String, String
     } catch (IOException e) {
       if (errCb != null) {
         errCb.accept(
-            "StrSplit_ParserFailure", "Parsing failed on '" + orig + "' (" + arg.context + ")");
```
Comment (pradh) [inline util/src/main/java/org/datacommons/util/McfParser.java:529]: Maybe here too we can add property?

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 2]
```diff
@@ -504,15 +528,20 @@ public static SchemaTerm parseSchemaTerm(String value, BiConsumer<String, String
...
+          }
+          errCb.accept(
+              "StrSplit_EmptyToken" + prop_suffix, "Empty value found (" + argContextString + ")");
         }
         return splits;
       }
       if (records.size() != 1) {
         if (errCb != null) {
           errCb.accept(
               "StrSplit_MultiToken",
```
Comment (pradh) [inline util/src/main/java/org/datacommons/util/McfParser.java:543]: Maybe here too we can add property?

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 3]
```diff
@@ -504,15 +528,20 @@ public static SchemaTerm parseSchemaTerm(String value, BiConsumer<String, String
       List<CSVRecord> records = parser.getRecords();
       if (records.isEmpty()) {
         if (errCb != null) {
-          errCb.accept("StrSplit_EmptyToken", "Empty value found (" + arg.context + ")");
+          String prop_suffix = "";
+          if (argContext != null) {
+            prop_suffix = "_" + argContext.prop;
+          }
+          errCb.accept(
+              "StrSplit_EmptyToken" + prop_suffix, "Empty value found (" + argContextString + ")");
```
Comment (pradh) [inline util/src/main/java/org/datacommons/util/McfParser.java:536]: In an existing test, maybe you could add a check like https://github.com/datacommonsorg/import/blob/master/util/src/test/java/org/datacommons/util/TmcfCsvParserTest.java#L46-L47 ?

**Review by pradh — COMMENTED** [review]
Thanks for the change!

File: util/src/main/java/org/datacommons/util/McfParser.java (hunk 1 above)
Comment (chejennifer) [reply util/src/main/java/org/datacommons/util/McfParser.java:529]: done

File: util/src/main/java/org/datacommons/util/McfParser.java (hunk 2 above)
Comment (chejennifer) [reply util/src/main/java/org/datacommons/util/McfParser.java]: done

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 4]
```diff
@@ -484,12 +484,36 @@ public static SchemaTerm parseSchemaTerm(String value, BiConsumer<String, String
     public char delimiter = ',';
     public boolean includeEmpty = false;
     public boolean stripEnclosingQuotes = true;
-    public String context;
+    public SplitAndStripArgContext context;
   }
+
+  public static final class SplitAndStripArgContext {
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply util/src/main/java/org/datacommons/util/McfParser.java:490]: will do the different approach in a separate pr

File: util/src/main/java/org/datacommons/util/McfParser.java (hunk 3 above)
Comment (chejennifer) [reply util/src/main/java/org/datacommons/util/McfParser.java:536]: done

**Review by pradh — APPROVED** [review]

**Pull request merged**